src/logistration/data/service.js

```diff
@@ -42,3 +42,25 @@ export async function login(creds) {
     success: data.success || false,
   };
 }
+
+
+export async function getThirdPartyAuthContext(urlParams) {
+  const requestConfig = {
+    headers: { 'Content-Type': 'application/x-www-form-urlencoded' },
+    isPublic: true,
+  };
+
+  const { data } = await getHttpClient()
+    .get(
+      `${getConfig().LMS_BASE_URL}/api/third_party_auth_context`,
+      { params: urlParams },
+      requestConfig,
+    )
+    .catch((e) => {
+      throw (e);
+    });
+
+  return {
+    thirdPartyAuthContext: data,
```

Comment:
You have to write a response transformer as well which would convert the response to camel case.